src/semver/Version.d

```diff
@@ -0,0 +1,139 @@
+/*******************************************************************************
+
+    Utility to simplify working with semantic versioning entities, including
+    parsing and comparison.
+
+*******************************************************************************/
+
+module semver.Version;
+
+/**
+    Describes SemVer aware version instance
+ **/
+struct Version
+{
+    /// major version number
+    int major;
+
+    /// minor version number
+    int minor;
+
+    /// patch version number
+    int patch;
+
+    /// original version string
+    string representation;
+
+    /**
+        Returns:
+            standard string representation of the version
+     **/
+    string toString ( ) const pure
+    {
+        return this.representation;
```

Comment:
`this.representation` will be invalid, if an instance is constructed manually (i.e. not via `fromString()`).